internal/olm/operator/registry/olm_resources.go

```diff
@@ -89,6 +89,14 @@ func withSecrets(secretNames ...string) func(*v1alpha1.CatalogSource) {
 	}
 }
 
+func withGrpcPodConfig(securityContextConfig string) func(*v1alpha1.CatalogSource) {
+	return func(cs *v1alpha1.CatalogSource) {
+		cs.Spec.GrpcPodConfig = &v1alpha1.GrpcPodConfig{
+			SecurityContextConfig: v1alpha1.SecurityConfig(securityContextConfig),
+		}
+	}
+}
```

Comment:
Function name should align with what we're passing into the function. Either pass in the full grpcPodConfig or rename (and refactor) to _just_ handle the securityContextConfig.

Suggestion showing the latter option:
```suggestion
func withGrpcPodSecurityContextConfig(securityContextConfig string) func(*v1alpha1.CatalogSource) {
	return func(cs *v1alpha1.CatalogSource) {
		if cs.Spec.GrpcPodConfig == nil {
			cs.Spec.GrpcPodConfig = &v1alpha1.GrpcPodConfig{}
		}
		cs.Spec.GrpcPodConfig.SecurityContextConfig = v1alpha1.SecurityConfig(securityContextConfig)
		}
	}
}
```

Reply:
Should be resolved as of https://github.com/operator-framework/operator-sdk/pull/6210/commits/1d2e1ed905b49260cb84f3be32cd05d0470d3736